code/src/components/Container.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import Counter from './Counter'
+import TodoList from './TodoList'
+import TodoForm from './TodoForm'
+import Header from './Header'
+
+const Container = () => {
+  const items = useSelector((store) => store.todos.items)
+
+  return (
+    <Body>
+      <Wrapper>
+        <Header />
+        <Counter display="remaining" items={items}/>
+        <TodoList items={items} complete={false}/>
+        <TodoForm />
+        <Counter display="finished" items={items}/>
+        <TodoList items={items} complete/>
```

Comment:
Great with the structure of displaying finished / unfinished todos at the top and the bottom of the application, a nice example of a reusable components for this kind of application!